apps/www/components/Pledge/CustomizePackage.js

```diff
@@ -425,13 +425,8 @@ class CustomizePackage extends Component {
           )
         }),
     )
-    const payMoreSuggestions = [
-      'DONATE',
-      'ABO_GIVE',
-      'ABO_GIVE_MONTHS',
-      'YEARLY_ABO',
-      'LESHA',
-    ].includes(pkg.name)
+
+    const payMoreSuggestions = pkg.options[0].payMoreSuggestion
```

Comment:
It would be best to invert this field in the code, so that payMoreSuggestion = true in the DB when the suggestion should be shown, (and not if should *not* be shown as it is now), but I wasn't sure how to do that best

Reply:
Yes, I (or we both) can have a look at that

Reply:
Wait, the old logic is if the condition in `payMoreSuggestions` is true, then the actual suggestions are `[]`????

Reply:
Yes, it doesn't make a lot of sense to me that it's written like this (hence why I was confused)